ova/Libraries/provision-opendistro.sh

```diff
@@ -0,0 +1,293 @@
+#!/bin/bash
+char="#"
+sys_type="yum"
+searchguard_version="1.8"
+resources_url=https://raw.githubusercontent.com/wazuh/wazuh-documentation/${BRANCH}
+
+logger() {
+
+    echo $1
+}
+
+startService() {
+
+    if [ -n "$(ps -e | egrep ^\ *1\ .*systemd$)" ]; then
+        systemctl daemon-reload 
+        systemctl enable $1.service 
+        systemctl start $1.service 
+        if [ "$?" != 0 ]; then
+            logger "${1^} could not be started."
+            exit 1;
+        else
+            logger "${1^} started"
+        fi
+    elif [ -n "$(ps -e | egrep ^\ *1\ .*init$)" ]; then
+        chkconfig $1 on 
+        service $1 start 
+        /etc/init.d/$1 start 
+        if [ "$?" != 0 ]; then
+            logger "${1^} could not be started."
+            exit 1;
+        else
+            logger "${1^} started"
+        fi
+    elif [ -x /etc/rc.d/init.d/$1 ] ; then
+        /etc/rc.d/init.d/$1 start 
+        if [ "$?" != 0 ]; then
+            logger "${1^} could not be started."
+            exit 1;
+        else
+            logger "${1^} started"
+        fi
+    else
+        logger "Error: ${1^} could not start. No service manager found on the system."
+        exit 1;
+    fi
+}
```

Comment:
```suggestion
startService() {

    service_name=$1

    if command -v systemctl > /dev/null 2>&1 && systemctl > /dev/null 2>&1; then
        systemctl daemon-reload 
        systemctl enable ${service_name}.service 
        systemctl start ${service_name}.service 
        if [ "$?" != 0 ]; then
            logger "${1^} could not be started."
            exit 1;
        else
            logger "${1^} started"
        fi
    elif command -v service > /dev/null 2>&1; then
        if command -v chkconfig > /dev/null 2>&1; then
            chkconfig ${service_name} on
        elif command -v update-rc.d > /dev/null 2>&1; then
            update-rc.d ${service_name} defaults
        fi
        service ${service_name} start 
        if [ "$?" != 0 ]; then
            logger "${1^} could not be started."
            exit 1;
        else
            logger "${1^} started"
        fi
    else
        logger "Error: ${1^} could not start. No service manager found on the system."
        exit 1;
    fi
}
```

Reply:
Done in 599737656bcaaa119a2c30c76c1760fa0590643a